docs/device_resources.md

```diff
@@ -0,0 +1,355 @@
+| OUT           | Out       |
+
+The BEL BLUT3 has 4 BEL pins:
+
+| BEL pin name | Direction |
+|--------------|-----------|
+| I0           | In        |
+| I1           | In        |
+| I2           | In        |
+| O            | Out       |
+
+The BEL A0 has 1 BEL pin:
+
+| BEL pin name | Direction |
+|--------------|-----------|
+| A0           | Out       |
+
+The BEL OUTMUX has 4 BEL pins:
+
+| BEL pin name | Direction |
+|--------------|-----------|
+| BLUT         | In        |
+| XOR          | In        |
+| ALUT         | In        |
+| OUT          | Out       |
+
+
+There are 12 site PIPs:
+
+| BEL name | In pin | Out pin |
+|----------|--------|---------|
+| BLUT3    | I0     | O       |
+| BLUT3    | I1     | O       |
+| BLUT3    | I2     | O       |
+| ALUT3    | I0     | O       |
+| ALUT3    | I1     | O       |
+| ALUT3    | I2     | O       |
+| FFMUX    | ALUT   | D       |
+| FFMUX    | XOR    | D       |
+| FFMUX    | BLUT   | D       |
+| OUTMUX   | ALUT   | OUT     |
+| OUTMUX   | XOR    | OUT     |
+| OUTMUX   | BLUT   | OUT     |
+
+
+The site wire BLUT3\_O has 4 BEL pins:
+
+| BEL Name | Pin  |
+|----------|------|
+| BLUT3    | O    |
+| CARRY    | SI   |
+| FFMUX    | BLUT |
+| OUTMUX   | BLUT |
+
+The site wire B0 has 2 BEL pins:
+
+| BEL Name | Pin |
+|----------|-----|
+| B0       | B0  |
+| BLUT3    | I0  |
+
+## Details
+
+### Net routing summary
+
+Each net start at the driver output/inout BEL pin.  The BEL pin will be
+connected to exactly 1 site wire.  If the net sink can be reached within the
+site, then the net can use site PIPs to reach a site wire connected to the
+input/inout BEL pin.
+
+If the net sink is in another site, then the net must first reach a site port
+input BEL pin using site PIPs to reach the site wire connected to the site
+port. From there the net leaves the site via the site port and is now on the
+first node via the site pin matching the site port.
+
+From there the net must use PIPs to expand to new nodes until arriving at a
+node attached to valid site pin for the sink.  This would be a site pin that
+is part of the same site that the sink BEL is part of, and that the site port
+wire can reach the sink BEL pin (via 0 or more site PIPs). The site can be
+entered via the site port corresponding to the site pin. The first site wire
+in the site will be the site wire attached to the output BEL pin of the site
+port.  From there site routing continues per above.
+
+![Wire and nodes](https://symbiflow.readthedocs.io/projects/arch-defs/en/latest/_images/rrgraph-wire.svg)
+
+### Tile Types and site types
+
+To reduce data duplication in the device schema, both tiles and sites have a
+type.  Most of the definition of the tile and site is in the type rather than
+repeated at each instance.  This does cause some more complicated
+indirection, so the following section provides some additional explanation
+here.
+
+#### Sites, site types and alternative site types
```

Comment:
As far as I get it, the alternative site types are a similar concept to the equivalent sites in VPR, is that correct?

And for instance, this also applies to some site types in the 7-series which can be implemented in different ways, e.g. the ILOGIC site can be implemented as a:
- ILOGICE3 (containing some routing BELs and FFs)
- ISERDESE2 (which is a single logic BEL)

Reply:
You are correct.  This point of this section is more to explain the how of the encoding, rather than the why. But yes, alternate site types is how ILOGICE3/ILOGICE2/ISERDESE2 site types are encoded.